libs/blocks/video/video.js

```diff
@@ -6,7 +6,7 @@ const ROOT_MARGIN = 1000;
 const loadVideo = (a) => {
   const { pathname, hash } = a;
   let videoPath = `.${pathname}`;
-  if (pathname.match('media_.*.mp4')) {
+  if (pathname.match('media_.*.(?:mp4|m4v)')) {
```

Comment:
can't we enforce character `\.` rather than using wildcard '.' ?
you are not using matching group down here so i would suggest not to use `?:`

Reply:
I agree with @npeltier, I think we could use something like `media_.*\.(mp4|m4v)`.

Or, looking at Nicolas' other suggestion, we may be able to get the values of the `video` auto blocks and build a `.mp4|.m4v)` string out of them. We would then just have to update _utils.js_ once and then everything trickles down automatically.

Reply:
how about this: if (pathname.match(/^\/media_.*\.(mp4|m4v)$/))

Reply:
this matches /mediawhatevermp4 and /mediaxwhateverm4v

Reply:
Adding some visuals to this, @npeltier is right:

- initial implementation, `media_.*.(?:mp4|m4v)`
<img width="609" alt="Screenshot 2023-12-14 at 15 36 16" src="https://github.com/adobecom/milo/assets/11267498/2e128a80-30e7-4eb7-8bdc-0b07eb179957">

- my suggestion from above, `media_.*\.(mp4|m4v)`
<img width="609" alt="Screenshot 2023-12-14 at 15 39 52" src="https://github.com/adobecom/milo/assets/11267498/32a85ee0-2b7c-45be-a103-baf69d65acfc">

- the latest suggestion is invalid, so it would just lead to an error